README.md

```diff
@@ -99,42 +101,46 @@ There are three ways to configure, in descending order of precedence:
 In your application, e.g.
 
     DockerClientConfig config = DockerClientConfig.createDefaultConfigBuilder()
-        .withVersion("1.16")
-        .withUri("https://my-docker-host.tld:2376")
-        .withUsername("dockeruser")
-        .withPassword("ilovedocker")
-        .withEmail("dockeruser@github.com")
-        .withServerAddress("https://index.docker.io/v1/")
-        .withDockerCertPath("/home/user/.docker")
+        .withDockerHost("tcp://my-docker-host.tld:2376")
+        .withDockerTlsVerify("1")
+        .withDockerCertPath("/home/user/.docker/certs")
+        .withDockerConfig("/home/user/.docker")
+        .withApiVersion("1.21")
+        .withRegistryUrl("https://index.docker.io/v1/")
+        .withRegistryUsername("dockeruser")
+        .withRegistryPassword("ilovedocker")
+        .withRegistryEmail("dockeruser@github.com")
```

Comment:
why do you need registry related things in client initialisation?
I.e. if you have 2 auth registries you should call auth cmd and only then push. 
I also not sure what will happen with parallel pushes...